docs/getting-started/install.md

```diff
@@ -23,69 +24,68 @@ A suitable python installation with `pip`. Minimum version requirement for Pytho
 
 1. Install Redis, an open-source, in-memory data store used by Flojoy:
 
-    ```bash
-    brew install redis
-    ```
+   ```bash
+   brew install redis
+   ```
 
    ([Redis installation reference](https://redis.io/docs/getting-started/installation/))
-    
-2. [Clone the repo](https://github.com/flojoy-io/studio) with `--recursive` argument as follows: 
-    
-    ```bash
-    git clone --recursive https://github.com/flojoy-io/studio.git
-    ```
-    
+
+2. [Clone the repo](https://github.com/flojoy-io/studio) with `--recursive` argument as follows:
+
+   ```bash
+   git clone --recursive https://github.com/flojoy-io/studio.git
+   ```
+
 3. Navigate into the project root:
-    
-    ```bash
-    cd studio
-    ```
-    
+
+   ```bash
+   cd studio
+   ```
+
 4. We highly recommend creating a Python virtual environment inside the `studio` folder by running the command:
-    
+
    ```bash
    python3.10 -m venv env
    ```
 
 5. Finally, run:
-    
-    ```bash
+   ```bash
    sh flojoy -v env
-    ```
-    
-  </TabItem>
+   ```
+
+</TabItem>
+
+ <TabItem value="windows" label="Windows">
 
-  <TabItem value="windows" label="Windows">
-    
 To install Flojoy on Windows:
 
 1. [Install Memurai](https://www.memurai.com/get-memurai), an alternative to Redis for Windows
-  
+
 2. [Update your PowerShell](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-windows) to at least >=v7.0.0
-  
-3. [Clone the repo](https://github.com/flojoy-io/studio) with `--recursive` argument as follows: 
-    
-    ```bash
-    git clone --recursive https://github.com/flojoy-io/studio.git
-    ```
-    
+
+3. [Clone the repo](https://github.com/flojoy-io/studio) with `--recursive` argument as follows:
+
+   ```bash
+   git clone --recursive https://github.com/flojoy-io/studio.git
+   ```
+
 4. Navigate into the project root:
-    
-    ```bash
-    cd studio
-    ```
-    
+
+   ```bash
+   cd studio
+   ```
+
 5. We highly recommend creating a Python virtual environment inside the `studio` folder by running the command:
-    
+
```

Comment:
Env is not recommended for Windows users. As we don't have support for that yet in powershell script.